doc/modules/multiclass.rst

```diff
@@ -1,9 +1,9 @@
 
 .. _multiclass:
 
-====================================
-Multiclass and multilabel algorithms
-====================================
+=====================
+Multiclass algorithms
```

Comment:
Multilabel was removed from the title but there's still a whole section about multilabel classification below